storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -1505,4 +1507,20 @@ public static StormTopology addVersions(StormTopology topology) {
         }
         return defaultsConf;
     }
+
+    public static <V> ArrayList<V> convertToArray(Map<Integer, V> srcMap, int start) {
+        Set<Integer> executorIds = srcMap.keySet();
```

Comment:
Let's use more generic variable name: ids would be enough.

Reply:
indeed.